packages/aws-rfdk/bin/stage-deadline.ts

```diff
@@ -34,6 +35,11 @@ while (n < args.length) {
       n++;
       dockerRecipesURI = args[n];
       break;
+    case '-r':
+    case '--release':
```

Comment:
release seems like a weird name for this. What do you think of --version instead?

Reply:
Finding a good name for the option is tricky. `--version`/`-v` is a pretty typical standard option that means "what is the version of the thing that I'm running?" So, in this context, it doesn't fit.

`--release`/`-r` is also a little off because it almost implies that you're doing a release, not acquiring one.

Reply:
Originally, I had envisioned a positional argument for the version once we had a way to discover and resolve version queries of Deadline.

```
stage-deadline [--output OUTPUT_DIR] DEADLINE_VER
```

I think it makes it simpler to use. For now, since there is no way to resolve versions of Deadline other than fully-qualified patch releases (e.g. `10.1.9.6`), we could fail on things that require look-ups like `10.1.9`.

We could leave the existing usage so we don't break existing users' scripting.

What do you think?

Reply:
Can we go with 'deadlineInstallerVersion' so as to match with 'deadlineInstallerURI' ?

Reply:
1) I can switch this to an optional positional argument easily enough, I'm not particularly tied to it being a named argument. For now it would just need the fully-qualified version, but that can be relaxed later.

2) How about `deadlineReleaseVersion` for the variable name? It really is more a release version; which happens to match the installer's version too.

Reply:
`deadlineReleaseVersion` sounds good to me.

Reply:
Thanks for incorporating the suggestion. Resolving